packages/p2p/src/components/buy-sell/filter-modal/filter-modal.scss

```diff
@@ -53,22 +53,15 @@
     }
 
     &__toggle {
-        align-items: center;
-        display: flex;
-        justify-items: flex-start;
-        transform: scale(0.5) translateX(-50%);
-
         &-button {
-            top: 3px;
-
-            &--on {
-                left: calc(100% - 4px);
-            }
+            margin-top: -1px;
```

Comment:
Can we use margin-bottom instead?

Reply:
I double checked it, it seems there was a bug during development. We don't need here margin.
deleted
thanks